charmhelpers/contrib/openstack/context.py

```diff
@@ -2577,6 +2580,22 @@ def cpu_mask(self):
                 mask = mask | 1 << core
         return format(mask, '#04x')
 
+    def pmd_cpu_mask(self):
```

Comment:
This looks like a `@classmethod`?  i.e. it doesn't seem to use any of the instance members of the class.  As a class method it would access `_parse_cpu_list()` via the `cls` method param (instead of `self`).  Thanks.

Reply:
updated PR with @classmethod and use of cls.

Reply:
Changed it back as it now calls cpu_mask().